### PR TITLE
Add Fabric platform (W.I.P)

### PR DESCRIPTION
**Pull request by Yu-Vitaqua-fer-Chronos**
Not ready for merging, this is hell to implement lol

### CODE REVIEW

File: Unlicense [hunk 1]
```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```
Comment (MercerK) [inline Unlicense]: Maybe Grakkit should use a MIT license instead?

Comment (Yu-Vitaqua-fer-Chronos) [reply Unlicense]: Why's this? Spacey said that he thinks the Unlicense is the best fit, but if you can give other reasons why not to use it, I'm all for it

Comment (MercerK) [reply Unlicense]: My primary reason is basically that I believe credit goes to where credit is due. Hard work went into this project just to make it work, and I believe those that created this wonderful project should forever get credited for their work. With the MIT License, you can still do a lot, but those that do work off of Grakkit in different forms will still end up crediting the original author(s). 

Outside of that, I personally feel that the "MIT" is the safer solution for licensing, After researching "unlicense" further, it seems a bit more concerning. One common bit that I've found is that Germany (maybe even France) doesn't recognize public domain licenses, which could potentially limit our audience.

Comment (Yu-Vitaqua-fer-Chronos) [reply Unlicense]: Ah fair enough, that's a reasonable concern

**Comment by Yu-Vitaqua-fer-Chronos** [comment]
Closing this because it's not being developed so

**Pull request closed without merging**